frontend/src/pages/PropertyDetails.jsx

```
@@ -429,6 +412,182 @@ const PropertyDetails = () => {
+                isEnabled={bookingDates.checkIn && bookingDates.checkOut && property.status === 'available' && user}
+                onClick={handleBooking}
+                user={user}
+            />
+            </div>
+        );
+    };
+
+    const DateInput = ({ label, selected, onChange, minDate, maxDate, excludeIntervals }) => (
+        <div>
+          <label className="block text-gray-700 mb-2">{label}</label>
+          <DatePicker
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)